src/ConfCrypt/Commands.hs

```diff
@@ -79,9 +81,9 @@ instance (Monad m, MonadDecrypt (ConfCryptM m key) key) => Command GetConfCrypt
     evaluate (GetConfCrypt name) = do
         (ccFile, ctx) <- ask
         let mParam = find ((==name) . paramName) (parameters ccFile)
-        traverse_ (decrypt ctx) mParam
+        traverse (decrypt ctx) $ maybeToList mParam
         where
-            decrypt ctx = tell . pure <=< decryptValue ctx . paramValue
+            decrypt ctx = pure <=< decryptValue ctx . paramValue
```

Comment:
Why does this use `pure <=<`? that's equivalent to the rhs of `<=<`

Reply:
That's right! Corrected in last commit.